plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/TimezoneRegistry.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.registry;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.ModelPreferences;
+import org.jkiss.dbeaver.model.preferences.DBPPreferenceStore;
+import org.jkiss.dbeaver.runtime.DBWorkbench;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.TimeZone;
+import java.util.stream.Collectors;
+
+public class TimezoneRegistry {
+
+    public static final String DEFAULT_VALUE = "Default";
+    public static String userDefaultTimezone = "";
+
+    private TimezoneRegistry() {
+    }
+
+    public static void setDefaultZone(@Nullable ZoneId id) {
+        DBPPreferenceStore preferenceStore = DBWorkbench.getPlatform().getPreferenceStore();
+        if (id != null) {
+            preferenceStore.setValue(ModelPreferences.CLIENT_TIMEZONE, id.getId());
+            TimeZone.setDefault(TimeZone.getTimeZone(id));
+            System.setProperty("user.timezone", id.getId());
+        } else {
+            preferenceStore.setToDefault(ModelPreferences.CLIENT_TIMEZONE);
+            TimeZone.setDefault(TimeZone.getTimeZone(userDefaultTimezone));
+            System.setProperty("user.timezone", userDefaultTimezone);
```

Comment:
The `user.timezone` property is only used within the `TimeZone` class. What's the point of changing it?